lib/puppet/type/rabbitmq_policy.rb

```diff
@@ -120,10 +120,15 @@ def validate_definition(definition)
       ha_sync_batch_size_val = definition['ha-sync-batch-size']
       raise ArgumentError, "Invalid ha-sync-batch-size value '#{ha_sync_batch_size_val}'" unless ha_sync_batch_size_val.to_i.to_s == ha_sync_batch_size_val
     end
-    if definition.key? 'delivery-limit' # rubocop:disable Style/GuardClause
+    if definition.key? 'delivery-limit'
       delivery_limit_val = definition['delivery-limit']
       raise ArgumentError, "Invalid delivery-limit value '#{delivery_limit_val}'" unless delivery_limit_val.to_i.to_s == delivery_limit_val
     end
+    if definition.key? 'initial-cluster-size'
```

Comment:
Took me a little to figure these failures out, but I think you can keep the code it was, and move the rubocop disablement to the last conditional, which will probably keep this most consistent.

Reply:
Oh nice, thanks, will do that